packages/tocco-util/src/units/addSystemUnit.js

```diff
@@ -0,0 +1,15 @@
+import _round from 'lodash/round'
+
+export const addSystemUnit = value => {
```

Comment:
i dont quite like the name. Wouldn't "shorten something" be a better name?

Reply:
I change to "condenseNumber".